skimage/_shared/tests/test_utils.py

```diff
@@ -347,3 +349,318 @@ def test_deprecate_func():
     )
     assert record[0].lineno == expected_lineno
     assert record[0].filename == __file__
+
+
+@deprecate_parameter("old1", start_version="0.10", stop_version="0.12")
+@deprecate_parameter("old0", start_version="0.10", stop_version="0.12")
+def _func_deprecated_params(arg0, old0=DEPRECATED, old1=DEPRECATED, arg1=None):
```

Comment:
```suggestion
def _func_deprecated_params(arg0, arg1=None, old0=DEPRECATED, old1=DEPRECATED):
```
It's fine since all `old0, old1, arg1` are kwargs, but I find it more readable (less cognitive load) with an 'intuitive' order. Or do you need this specific order for testing some edge cases? I may be missing something...

Reply:
I agree, but I used this order on purpose to test that the decorators play well with arguments before and after deprecated parameters. Ordering it like you suggest wouldn't cover the latter case anymore.

Reply:
Perfect, I suspected something like this... :wink: Please ignore my suggestion then.